src/GraphQL/DocumentWriter.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphQL
+{
+    public interface IDocumentWriter
```

Comment:
I propose to split this file into two and rename.

Reply:
It should be split into two files now.